webpack-modules/webpack.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+    entry: {
+        background_scripts: "./background_scripts/background.js",
+        popup: "./popup/left-pad.js"
+    },
+    output: {
+        path: "dist",
```

Comment:
Is "dist" a convention for webpack output? I thought something like "addon" might be more obvious.